src/main/java/mate/academy/spring/validation/EmailValidator.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.spring.validation;
+
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+    private static Pattern REGEX = Pattern.compile(
```

Comment:
This is Pattern. Previously you had constant with regex, but named it PATTERN. Do not confuse these concepts)
```suggestion
    private static Pattern EMAIL_PATTREN = Pattern.compile(
```